Maple2.Server.Game/PacketHandlers/ItemRepackHandler.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Linq;
+using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Packets;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class ItemRepackHandler : PacketHandler<GameSession> {
+    public override RecvOp OpCode => RecvOp.ItemRepack;
+
+    private enum Command : byte {
+        Commit = 1,
+    }
+
+    public override void Handle(GameSession session, IByteReader packet) {
+        var command = packet.Read<Command>();
+        switch (command) {
+            case Command.Commit:
+                HandleCommit(session, packet);
+                return;
+        }
+    }
+
+    private static void HandleCommit(GameSession session, IByteReader packet) {
+        long scrollUid = packet.ReadLong();
+        long targetItemUid = packet.ReadLong();
+
+        lock (session.Item) {
+            Item? scroll = session.Item.Inventory.Get(scrollUid);
+            if (scroll == null) {
+                session.Send(ItemRepackPacket.Error(ItemRepackError.s_item_repacking_scroll_error_invalid_scroll));
+                return;
+            }
+
+            Item? targetItem = session.Item.Inventory.Get(targetItemUid);
+            if (targetItem == null || targetItem.Binding != null) {
```

Comment:
seems like we should just check `targetItem.Transfer != null` here as well to simplify logic below.

Reply:
Do you mean it should be 
`if (targetItem == null || targetItem.Transfer != null || targetItem.Binding != null)` ?
I think I'd have to put this in another if statement or have the check on Binding in another statement otherwise it would allow it to pass through due to Binding

Reply:
i think it should be:
```cs
if (targetItem == null || targetItem.Transfer == null || targetItem.Binding != null)
```